docs/guides/setup_concord_with_env.md

```diff
@@ -0,0 +1,45 @@
+# Setting up with env variables
+
+## What is an "env variable"?
+
+Env variables are values that are dynamic-named that can affect how processes behave in case their values are edited. It can store values like paths, names, ints, and anything (except for files). Widely used in many applications.
+
+## JSON fields vs env variables
+
+Even JSON is widely used by applications for storing data, it is NOT good for storing sensitive data. And **env variables are much easier to get** than JSON fields since you can use very straightforward functions like `getenv`, while in **JSON you would require third-party libraries** such [jsmn-find](https://github.com/lcsmuller/jsmn-find) to read stored data.
+
+## Setting an env variable
+
+There are **multiple ways of setting env variables**, but **we don't want all the users in the machine to know the values of the env variables**, so we are going to stick with setting them into a `.sh` file that runs the bot (if using Concord or any type of program).
+
+First, **create a file** with whatever name you want, for example, `foo.sh`. Then, **put in the file the following content** (`myBot` is the file with the binary result of your bot):
+
+```sh
+#!/bin/csh
+setenv ENV_VARIABLE_NAME "The value of the env variable"
+
+./myBot
+```
```

Comment:
"Please note that this only applies to the BSD C Shell. If you are using the Korn shell or Bash, you will need to use something like this:
```
#!/bin/sh
export ENV_VARIABLE_NAME="The value of the env variable"
./myBot
```
"

Reply:
You meant to replace `First, **create a file** with whatever name you want, for example, `foo.sh`. Then, **put in the file the following content** (`myBot` is the file with the binary result of your bot):` with it? Because it doesn't say that you need to create a file and put this in, or you are trying to say to use this as a command and not as content in a file?

Reply:
What I meant by this was add a Korn Shell style example too, but, I think I'm getting tripped up on what you're mentioning here. All of those can be executed as commands, yeah.